android/src/main/java/com/rcttabview/RCTTabView.kt

```diff
@@ -140,4 +143,28 @@ class ReactBottomNavigationView(context: Context) : BottomNavigationView(context
     super.onDetachedFromWindow()
     isAnimating = false
   }
+
+  fun setBarTintColor(color: Int?) {
+    updateBackgroundColors(color)
+  }
```

Comment:
nit: You can just move the function body here, no need for `updateBackgroundColors` as its not reused anywhere